kernel/src/main/java/org/kframework/parser/concrete2kore/disambiguation/AddEmptyLists.java

```diff
@@ -63,6 +63,24 @@ public AddEmptyLists(Module m, Sort expectedSort) {
         this.expectedSort = expectedSort;
     }
 
+    private Tuple2<Either<Set<KEMException>, Term>, Set<KEMException>> superApply(TermCons tc) {
```

Comment:
The name of this function looks funny to me. Could you at least add some documentation?